src/app/components/mnoe-audit-events-list/mnoe-audit-events-list.html

```diff
@@ -22,8 +22,8 @@
         </tbody>
         <tbody ng-show="!$ctrl.events.loading">
           <tr ng-repeat="event in $ctrl.events.list">
-            <td title="{{ event.created_at | amDateFormat: 'DD/MM/YYYY, h:mm:ss a' }}">
-              {{event.created_at | amCalendar }}
+            <td title="{{ event.created_at | amDateFormat: 'L h:mm:ss a' }}">
```

Comment:
Change to `'L LTS'`, for country using 24 hours clock (eg France) we'll get: `28/10/2018 16:25:21`
For country using 12  hours clock we'll get: `10/28/2018 4:25:21 PM`

Reply:
👍